### ISSUE TITLE
legacy file issue (already tried existing solve)

### ISSUE THREAD

**Issue by apartmentcat** [OpenShot/openshot-qt#5131]
**Describe the bug:**
Hi, so I created a project and have worked on it a few times fine. Then when I tried to open it later in the day, I got a "failed to load legacy project file" error message. My partner suggested reverting back to version 2.6 but the same issue occurred. I then re-installed v 3.0 (from the latest daily build page) and still had the same issue. I had also tried deleting the ".openshot_qt" folder earlier on as per some other instructions I read. 
I already tried following the advice in the Reddit thread to go to the recovery folder and retrieve the backup files from there, but the folder was empty. This is the thread I was referencing:
https://www.reddit.com/r/OpenShot/comments/yxckul/openshot_erased_my_video/

**Steps to reproduce the behavior:**
Both ways of opening the project file result in the same error message:
1. Boot up Openshot program. 2. Go to File menu > open > select relevant project file
1. double-click on relevant project file.

**Expected behavior:**
I wanted to open and work on my project.

**System Details:**
- OpenShot Version 3.0 ALSO 2.6
- Operating System: Windows 10

**Log Files:**
There seems to be no relevant log file in the relevant folder. The only one I can see is titled "libopenshot.log"

**Screenshots:** (Optional)
Screenshot of error message:
![legfile](https://user-images.githubusercontent.com/124847296/217688608-0374f622-44d7-4b59-b0d4-6b57804c7cc3.GIF)

**Comment by Colorjet3** [comment]
Hello @apartmentcat I need some details before we can move forward:

1.  Were you running v2.6.1 before upgrading to v3.0.0?
2.  Did you create your project with v2.6.1 or v3.0.0?
3.  Did you make a backup of your c:\users\username\.openshot_qt folder?  If yes, was it made before you upgraded to v3.0.0 (assuming you did upgrade)?
4. How complex is your project?  Is it very large, many effects, many animation, etc?
5. If you upgraded to v3.0.0, did you uninstall v2.6.1 first?
6. Go to Add or Remove Programs and search for Openshot.  How many entries do you see of Openshot?

Once we have this information we can formulate a plan of attack to get you back to working condition.

**Comment by apartmentcat** [comment]
Hi @Colorjet3 , many thanks for your help!

    Were you running v2.6.1 before upgrading to v3.0.0?
Yes, I had 2.6.1 installed previously. I upgraded to v3.0 before starting the project. Then I had an issue with with audio running too fast. As per advice I found in the Reddit thread, I uninstalled it and replaced it with the latest dev daily build (3rd February). I deleted the project I had previously created and started a new one with that version. It worked fine for about 3 sessions of using the program, before the error message came up. 

    Did you create your project with v2.6.1 or v3.0.0?
I created the project with v3.0. (dev daily build as per above)
My husband suggested uninstalling v3.0 and re-installing v2.6 and trying to open the project with that. I had the same error message. So then I uninstalled v2.6 and re-installed v3.0. Still had the same message. ( I hope that didn't make matters worse...)

    Did you make a backup of your c:\users\username.openshot_qt folder? If yes, was it made before you upgraded to v3.0.0 (assuming you did upgrade)?
No. (I will make sure to do that in future!)
When I was trying to solve the issue with audio running too fast, I saw advice on the Reddit thread to delete that folder before installing the new version. So that's what I did. But I also deleted the project I created under that version, so I don't think that's the issue? 

    How complex is your project? Is it very large, many effects, many animation, etc?
I would say it's not very complex -- it's approx 20 minutes long, and has approx. 4 titles, 3 short audio effects, no animations.  

    If you upgraded to v3.0.0, did you uninstall v2.6.1 first?
Yes.

    Go to Add or Remove Programs and search for Openshot. How many entries do you see of Openshot?
I see one version -- 3.0.0

**Comment by Colorjet3** [comment]
Hello @apartmentcat Thank you so much for all the answers to my questions.  I think the best approach will be to recreate your project with the latest DEV Daily Build as it is difficult to debug with going back and forth between versions.  If you are ok with this I would perform the following:

1. Close Openshot. 
2. Uninstall v3.0.0 via the Add or Remove Programs.
3. Make a backup of you current c:\users\username\.openshot_qt folder.  Most likely you will not need it but just in case you have created other projects that might be affected by these steps.
4. Delete c:\users\username\.openshot_qt folder.
5. Check "c:\program files\Openshot Video Editor" folder.  If it exists, delete it.  You will need administrative rights to perform this.
6. Check "c:\program files(x86)\Openshot Video Editor" folder (this is the 32bit just in case you installed that as well).  If it exists, delete it.  You will need administrative rights to perform this.
7. Go to openshot.org/download/#daily and download the latest DEV DAILY build available.
8. Install it.
9. Recreate your project.  Save often for now until you feel comfortable that things are working properly.

We can then start with a clean slate and debug further if needed.

**Comment by apartmentcat** [comment]
Many thanks @Colorjet3 !
It's a little disappointing that I can't retrieve the work I've already done, but I guess it's a lesson to think twice before blindly following my husband's advice in the future, lol.
Cheers and have a good weekend. :)

**Comment by Colorjet3** [comment]
Thank you @apartmentcat and for the laugh.  You have a great week/end as well.
